state/factory/workingset.go

```diff
@@ -244,7 +244,18 @@ func (ws *workingSet) State(s interface{}, opts ...protocol.StateOption) (uint64
 }
 
 func (ws *workingSet) States(opts ...protocol.StateOption) (uint64, state.Iterator, error) {
-	return ws.statesFunc(opts...)
```

Comment:
think no need to change here, still pass the opts and let downstack funcs process